OBIII.service

```diff
@@ -4,6 +4,7 @@ After=multi-user.target
 
 [Service]
 Type=simple
+User=root
```

Comment:
Let's not push this until we can find a workaround for running this as root, if possible.

Reply:
cefcb7996f0587308d8e2bd41ae5b0b3e2352b9a